<launch>
  <node name="cope" pkg="cope" type="cope_node.py" output="screen">    
    <param name="model_path" value="/stefan/PyraPoseAF/models/real_zed_d435_center_tra05.h5" />
    <param name="meshes_path" value="/stefan/meshes" />
    <param name="image_topic" value="/hsrb/head_rgbd_sensor/rgb/image_raw" />
    <param name="detection_threshold" value="0.5" type="double" />
    <param name="icp_threshold" value="0.15" type="double" />
    <param name="service_call" value="/PyraPose/return_poses" />
  </node>
</launch>
